### DISCUSSION TITLE
Definitions, Conditionals, and Realizables: A Methodological Concern

### DISCUSSION THREAD

**Discussion by gregfowlerphd** [JisooSeo/101-WG-Discussions#14] [General]
**_The Old Objection_**

This is a bit of a follow-up to my earlier post "Definitions and Realizable Entities in BFO". There I presented an objection to a definition of "manufacturing tool" I found in a paper by Barry about the Industrial Ontologies Foundry:

>manufacturing tool(x) ≡ material-entity(x) & ∃f (BFO:function(f) & has- function(x, f) & ∀y(realizes(y, f) → manufacturing process (y)))

My objection was that given this definition, any material entity with an unrealized function--e.g., an unused kitchen knife, baby monitor, or action figure--trivially counts as a manufacturing tool.

You might remember from the WG meeting a few weeks ago that when Carter presented definitions from OccO, I raised a similar worry about the definition of "occupation role":

>A role borne by a human that, if realized, is realized when the bearer provides labor or services in exchange for compensation as specified by some deontic declaration.

My contention was that this definition was incorrect because it has the consequence that any role borne by a human that is never realized is (trivially) an occupation role.

**_John's Response_**

Since I'd seen several other definitions involving realizable entities in BFO-conformant ontologies that were subject to analogous concerns, I decided to email John (@johnbeve) about the issue. As Carter had anticipated in the WG meeting, John's response was that the "if" in the relevant definitions isn't the material conditional but is instead intended to express a modal notion.

This seems like a fine response to the concern: Where the conditional isn't material, my objection can't get off the ground. (Perhaps ironically, John and I agree that the response doesn't work in the case of Barry's FOL definition of "manufacturing tool", because the conditional there does appear to be material.) However, I now have new objection to the relevant definitions, one that I guess could be broadly classified as methodological.

**_The New Objection_**

Why not make the modal notion--the one that the "if" in the relevant definitions is supposed to express--explicit?

Defining terms in an ontology is, at least in part, an act of communication--in particular, with users of the ontology. Now in general, with some exceptions (e.g., a company issuing an elusive memo about changes to company policy in order to preserve employee morale), one desideratum of communication is clarity. This is especially so in intellectual pursuits, and although ontology development has practical aims, it would at least be classed, I think, as quasi-intellectual.

(Moreover, if you don't mind a bit of appeal to authority, although Barry hasn’t written at length about the importance of clarity in his publications on constructing definitions in ontologies, he has mentioned it a number of times. So has Andrew Spear.)

It seems to me, however, that using a bare "if" in the relevant definitions when what is intended is a modal notion obscures rather than clarifies.

Of course, these considerations only constitute a prima facie case against the use of a bare "if". It might be, for example, that spelling out the intended modal notion would make the definitions unwieldy or technical, and that might constitute sufficient reason to continue to use a bare "if" even if it results in a lack of clarity. To see whether such countervailing considerations apply, though, we need to consider how the relevant definitions might be modified to make explicit the intended modal notions.

**_A Tentative Proposal_**

My tentative(!) proposal, as applied to a particular example, is to take the definition of "occupation role" presented above

>A role borne by a human that, if realized, is realized when the bearer provides labor or services in exchange for compensation as specified by some deontic declaration.

and change it to

>A role borne by a human that, to be realized, must be realized by the bearer providing labor or services in exchange for compensation as specified by some deontic declaration.

I don't think this is particularly unwieldy or technical, and as far as I can see, no similar countervailing considerations tell against it. It is, however, tentative, since I'm not entirely sure whether it captures the modal notion intended in the original. Perhaps John can weigh in.

**Comment by gregfowlerphd** [comment]
Really hope the lack of replies doesn't indicate that my post was incoherent or anything. ;)

**Reply by JisooSeo** [reply]
Ah sorry! I have been busy with a paper. I will be able to reply in couple of days! Thank you so much for your posts 😄

**Reply by gregfowlerphd** [reply]
Lol! No worries, Jisoo! 😊

Also, I like posting here. I think it would be even better if more folks from the WG participated.

**Reply by JisooSeo** [reply]
True! I should put more time into developing this repo. I will post this thread on Slack if you don't mind!

**Reply by gregfowlerphd** [reply]
Sure, that would be fine. 😊
